stl/inc/yvals_core.h

```diff
@@ -470,6 +471,13 @@
 #define _CONSTEXPR20 inline
 #endif // ^^^ inline (not constexpr) in C++17 and earlier ^^^
 
+// Functions that became constexpr in C++20, and requires is_constant_evaluated
+#ifdef __cpp_lib_is_constant_evaluated
+#define _CONSTEXPR20_ICE constexpr
+#else // ^^^ constexpr witn is_constant_evaluated / vvv inline (not constexpr) without is_constant_evaluated vvv
+#define _CONSTEXPR20_ICE inline

```

Comment:
There doesn't seem to be a reason to make this case `inline`?

Reply:
Make `inline` same as `_CONSTEXPR20`

Reply:
That's because `_CONSTEXPR20` is intended to be used with ordinary functions; I believe all the intended use cases for this are templates?

Reply:
In this pr, yes.

Reply:
Remove `inline`